tests/unittest/utils/util.py

```
@@ -451,3 +456,32 @@ def check_accuracy(a, b, atol, rtol, percent):
+@contextmanager
+def try_expose_error_in_ray(error_type: type[Exception]):
+
+    if os.environ.get("TLLM_DISABLE_MPI") != "1":
+        yield
+        return
+
+    try:
+        yield
+    except (ray.exceptions.RayActorError, ray.exceptions.RayTaskError) as e:
+        if hasattr(e, 'cause') and e.cause is not None:
+            raise e.cause
```

Ruff (0.13.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)